app/views/account_home/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<% content_for :title, t("account.your_account.heading") %>
+<% content_for :location, "your-account" %>
+<% content_for :account_navigation do %>
+  <%= render "account-navigation", page_is: yield(:location) %>
+<% end %>
+<% content_for :body_classes, "account" %>
+
+<div class="govuk-grid-row govuk-main-wrapper">
+  <div class="govuk-grid-column-one-third">
+    <%= render "account-navigation", page_is: yield(:location) %>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <main id="content" role="main">
+      <%= render "user-reminders" %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: yield(:title),
+        heading_level: 1,
+        font_size: "l",
+        margin_bottom: 4,
+      } %>
+      <% if @user["email"] %>
+        <p class="govuk-body accounts-your-account__email">
+          <%= @user["email"] %>
+        </p>
+      <% end %>
+      <% if @user.dig("services", "transition_checker") == "yes" %>
+        <p class="govuk-body">You've used the transition checker, so we should put the non-empty service card here.</p>
```

Comment:
This line was just a placeholder, it should be removed

Reply:
Thanks, I definitely removed that at one point, it probably re-added itself when I did a rebase/merge 🤦🏻‍♀️